### DISCUSSION TITLE
[crowdfunding] Kanban board

### DISCUSSION THREAD

**Discussion by apcloic** [YetiForceCompany/YetiForceCRM#13928] [Wishlist]
Hi,

Would be really nice to have kanban board next to gantt.
#9028
#10894

As @bpabiszczak wrote in the following comment, the estimated cost is 1500€
https://github.com/YetiForceCompany/YetiForceCRM/issues/9028#issuecomment-600346037

I can contribute with 100€.
Anyone interested in crowdfunding this feature ?

**Comment by simple-AMS** [comment]
@apcloic I'm down to assist with crowd funding. We need more to jump on board with this idea.
